doc/source/development/contributing_codebase.rst

```diff
@@ -807,8 +804,57 @@ across the number of specified cores or ``auto`` to utilize all the available co
    # Utilizes all available cores
    pytest -n auto pandas
 
-This can significantly reduce the time it takes to locally run tests before
-submitting a pull request.
+If you'd like to speed things along further a more advanced use of this
+command would look like this
+
+.. code-block:: bash
+
+    pytest pandas -n 4 -m "not slow and not network and not db and not single_cpu" -r sxX
+
+In addition to the multithreaded performance increase this improves test
+speed by skipping some tests using the ``-m`` mark flag:
+
+- slow: any test taking long (think seconds rather than milliseconds)
+- network: tests requiring network connectivity
+- db: tests requiring a database (mysql or postgres)
+- single_cpu: tests that should run on a single cpu only
+
+You might want to enable the following option if it's relevant for you:
+
+- arm_slow: any test taking long on arm64 architecture
+
+These markers are defined `in this toml file <https://github.com/pandas-dev/pandas/blob/55dc32437ea43a238975439ddb6c9dda81b33020/pyproject.toml>`_
```

Comment:
Last nitpick: should we link here to the file on "main"? (since you mention you can go there to check if new ones have been created. But if we point to a fixed one in time, that will never be the case)

Reply:
Good catch – thank you!